src/app/Controllers/talking.component.ts

```diff
@@ -153,6 +158,32 @@ export class TalkingComponent implements OnInit, OnDestroy {
                             (<HTMLElement>document.querySelector('#scrollDown')).style.bottom = inputElement.scrollHeight + 46 + 'px';
                         }
                     });
+
+                    if ( !/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)) {
+                        inputElement.focus();
+                    }
+
+                    return this.conversationApiService.ConversationDetail(this.conversationID);
+                }),
+                map(t => t.value)
+            )
+            .subscribe(conversation => {
+                if (!this.uploadService.talkingDestroyed) {
```

Comment:
It seems that our uploader is not working properly for the progress bar disappears.

Reply:
oh

Reply:
This is not a serious problem. 
There is no need to fix it.